libnavigation-util/src/main/java/com/mapbox/navigation/utils/thread/WorkThreadHandler.kt

```diff
@@ -0,0 +1,63 @@
+package com.mapbox.navigation.utils.thread
+
+import android.os.Handler
+import android.os.HandlerThread
+import com.mapbox.navigation.utils.extensions.quitSafelySupport
+import timber.log.Timber
+
+class WorkThreadHandler(
+    private val handleThreadName: String = HANDLE_THREAD_NAME
+) : ThreadHandler {
+
+    private lateinit var handlerThread: HandlerThread
+    lateinit var handler: Handler
+
+    override var isStarted: Boolean = false
+
+    override fun post(task: () -> Unit) {
+        if (!isStarted) {
+            start()
+        }
+        handler.post {
+            task.invoke()
+            stop()
+        }
+    }
+
+    override fun postDelayed(task: () -> Unit, delayMillis: Long) {
+        if (!isStarted) {
```

Comment:
why did you change this behaviour?

Reply:
To avoid necessity of call `start()` and `stop()` methods before calling `post()` and `postDelayed()` methods

Reply:
that was the highly important path of interface, why you think we have `start` and `stop` methods? 🤦‍♂ 
Please next time mention people that were responsible for feature that might be changed